### ISSUE TITLE
Accessibility - Form missing label

### ISSUE THREAD

**Issue by ghost** [groundhoggwp/holler-box#20]
Enjoying the plugin so far, nice and neat.

It appears as though the forms are not using labels, this is throwing errors during WCAG compliance testing.
Which is likely making it difficult for non-sighted users to navigate popups as well.

**Comment by scottopolis** [comment]
Hi, I'm not familiar with that test. Can you give me more information?

**Comment by ghost** [comment]
Using the following automated tool: https://achecker.ca/checker/index.php

It checks markup for Web Content Accessibility Guidelines (WCAG), in some provinces (i don't know about states) having at least AA compliance is a government mandate.
